### ISSUE TITLE
unoconvert standalone use

### ISSUE THREAD

**Issue by ltybenet** [unoconv/unoserver#142]
3.0 breaks `unoconvert` standalone usage
```
ImportError: Could not find the 'uno' library. This package must be installed with a Python installation that has a 'uno' library. This typically means you should install it with the same Python executable as your Libreoffice installation uses.
```
#29 Already supported

for standalone use, I have reverted to version 2.2.2

**Comment by regebro** [comment]
Could you include a traceback?

**Comment by regebro** [comment]
Never mind I see it now.

**Comment by regebro** [comment]
Fixed in 3.0.1

**Comment by regebro** [comment]
Are you running unoserver with the system python?
